main.js

```diff
@@ -0,0 +1,119 @@
+const submitButton = document.querySelector('.submit-btn');
+
+submitButton.addEventListener('click', function(){
+  const pageHr = document.createElement('hr')
+  const name = document.querySelector('.input-name').value;
+  const post = document.querySelector('.input-post').value;
+  const postSection = document.querySelector('.post-section');
+  const form = document.querySelector('form');
+
+  // Creating post div
+  let newPost = document.createElement('div');
+  newPost.style.textAlign = 'center';
+  newPost.style.marginBottom = '10px';
+  
+  // Creating post text
+  const newPostText = document.createElement('p');
+  const newPostContent = document.createTextNode(post);
+  newPostText.appendChild(newPostContent);
+  
+  // Creating name text
+  const newName = document.createElement('p');
+  const newNameText = document.createTextNode('By: ' + name);
+  newName.appendChild(newNameText);
+  
+  // Creating button div
+  let newButton = document.createElement('div');
+  newButton.style.textAlign = 'center';
+  newButton.style.marginBottom = '10px';
+
+  // Creating remove button
+  let rmButton = document.createElement('button');
+  rmButton.style.marginBottom = "20px";
+  rmButton.style.textAlign = "center";
+  rmButton.style.fontSize = "10px";
+  rmButton.appendChild(document.createTextNode("Remove"));
+  rmButton.addEventListener('click', function() {
+    postSection.removeChild(newPost);
+    postSection.removeChild(pageHr);
+  });
+
+  // Creating Comment button
+  let commentButton = document.createElement('button');
+  commentButton.style.fontSize = '10px';
+  commentButton.appendChild(document.createTextNode('Comment'));
+  commentButton.addEventListener('click', function() {
+    // Open the comment form below the post
+    createCommentForm(newPost);
+  });
+
+  newButton.appendChild(rmButton);
+  newButton.appendChild(commentButton);
+
+  // Creating a section for comments
+  let commentSection = document.createElement('div');
+  commentSection.className = 'comment-section';
+  commentSection.style.textAlign = 'center';
+  commentSection.style.marginTop = '10px';
+
+  
+  if(name === '' || post === ''){
```

Comment:
name is a vague variable name - recommend using more detailed names so people reading the code know exactly what is stored in here